content/en/blog/_posts/2021-12-13-security-profiles-operator-v0.4.0/index.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: blog
+title: "What's new in Security Profiles Operator v0.4.0"
+date: 2021-12-13
```

Comment:
if we can, let's pick a date that doesn't conflict with any of the post-release articles (we leave planned gaps to make that feasible).
Is the release date for SPO 0.4.0 already set?

Reply:
No, we just decided to do it at some point next week :)

Reply:
The post-release feature blogs are scheduled up til 12/16. What do you think about publishing this on 12/17?

Reply:
Sounds good to me, thank you!